static/app/views/performance/newTraceDetails/traceDrawer/details/issues/issues.tsx

```diff
@@ -135,7 +136,11 @@ function Issue(props: IssueProps) {
       </AssineeWrapper>
     </StyledPanelItem>
   ) : isError ? (
-    <LoadingError message={t('Failed to fetch issue')} />
+    <LoadingError
+      message={
+        error.status === 404 ? t('This issue was deleted') : t('Failed to fetch issue')
```

Comment:
Couldn't this also be a 404 because the issue never existed?

Reply:
[at this point](https://github.com/getsentry/sentry/blob/aa355a40d8b6b7b257296fad11530f50c242ff3b/static/app/views/performance/newTraceDetails/traceDrawer/details/issues/issues.tsx#L199-L201), the issue had to have existed before.

Reply:
Ah ok gotcha - thank you for clarifying 
:)

Reply:
no problem... to give you more information... the issue is returned by the endpoint https://pri-sentry.dev.getsentry.net:7999/region/de/api/0/organizations/pri-sentry/events-trace/558ae4f960134ccb9256f884266816ad/?limit=10000&timestamp=1727700756&useSpans=1

Reply:
to give you more context. the issue is coming from here https://pri-sentry.dev.getsentry.net:7999/region/de/api/0/organizations/pri-sentry/events-trace/558ae4f960134ccb9256f884266816ad/?limit=10000&timestamp=1727700756&useSpans=1 . In the `orphan_errors` property. Now it seems to be gone... interesting 🧐

Reply:
it is probably deleted from there after a few minutes